doc/getting_started/installation_win.rst

```diff
@@ -17,29 +17,39 @@ Windows system with the latest updates installed.
 
```

Comment:
Can we perhaps add a reference to `host_setup` so that users that were linked to this page directly can also go back and see the full installation picture? Also, the `This section describes how to configure your development environment and to build Zephyr applications ` is no longer accurate, since we now have additional instructions in the main Getting Started page.